### ISSUE TITLE
Console error: Refused to create a worker

### ISSUE THREAD

**Issue by chtenb** [magjac/graphviz-visual-editor#235]
I'm seeing errors in my dev console of the form:

[Report Only] Refused to create a worker from 'blob:http://magjac.com/dc63c696-2997-4d9b-adc6-b218f78ac6e3' because it violates the following Content Security Policy directive: "worker-src 'none'".

Not sure what concrete problems they cause, but it sounds unintentional.

**Comment by magjac** [comment]
Sorry for the delay. What platform and browser are you on?

**Comment by chtenb** [comment]
Windows/Chrome

**Comment by magjac** [comment]
Sorry for the even more delay. I cannot reproduce this on Windows 10 with Chrome version 129.0.6668.71 when accessing https://magjac.com/graphviz-visual-editor/ or http://magjac.com/graphviz-visual-editor/.

Can you still reproduce this?

**Comment by chtenb** [comment]
Yes. After some clicking around I found that the middle mouse button produced another error:

![image](https://github.com/user-attachments/assets/bde5b055-38c6-45e2-ae98-49df1bb91ff9)

**Comment by magjac** [comment]
Thanks. Which version of Windows and which version of Chrome?

The other error is probably https://github.com/magjac/graphviz-visual-editor/issues/251.

**Comment by chtenb** [comment]
129.0.6668.89 on windows 11

**Comment by magjac** [comment]
I've added `Content-Security-Policy: worker-src blob:;` on the server. Can you try again? I suspect it has to do with Windows 11, but I cannot test that since I don't have such a machine.

**Comment by chtenb** [comment]
Still same error message

**Comment by magjac** [comment]
OK. Thanks. I think I'll have to put this issue on the back burner until I can get my hands on a Windows 11 machine.
